src/Messages.sol

```diff
@@ -159,8 +159,8 @@ library ConnectorMessages {
      * 47-78: amount (uint256 = 32 bytes)
      * 
      */
-    function formatTransfer(uint64 poolId, bytes16 trancheId, address user, uint256 amount) internal pure returns (bytes memory) {
-        return abi.encodePacked(uint8(Call.Transfer), poolId, trancheId, user, amount);
+    function formatTransfer(uint64 poolId, bytes16 trancheId, address user, uint256 amount, uint32 destinationDomain) internal pure returns (bytes memory) {
+        return abi.encodePacked(uint8(Call.Transfer), poolId, trancheId, user, amount, destinationDomain);
     }
 
     function isTransfer(bytes29 _msg) internal pure returns (bool) {
```

Comment:
Can't leave a comment there but we are forgetting to parse the domain from a `Transfer` message. However, I am not sure I understand how useful it is to have that domain at the Connector contract level? 🤔

Reply:
Yah, I'm not sure I understand why we're sending the domain to the connector. Seems like it should only be the router that cares about that domain for Cent -> EVM messages, right?

Reply:
@NunoAlexandre I propose we leave it out until we can figure out a reason to have it there.

Reply:
@AStox let's get @offerijns pov here since he had suggested adding the field to the message.

Reply:
We need it for EVM > EVM transfers in the future, since they will go through Cent Chain and thus will need to know the target domain. While we don't need to implement it today, it would be a shame if ew have to change the Transfer message format later.

Reply:
@offerijns what about the `parseTransfer()` function. No reason to be sending the domain to the connector for an incoming message, right?

Reply:
Makes sense @offerijns!

And that also makes sense @AStox

Reply:
Do you want to have two different `Transfer` messages, one that goes to Cent Chain and one that goes to Connectors? Or do I misunderstand?

Reply:
@offerijns i think Adam means that there is no point currently in actually parsing the domain field from the Transfer message. I think this is fine but it's also fine to already have that code in place and tested. Wdyt?

Reply:
Yah I thought Nuno's original question here was why aren't we parsing the domain from the `Transfer` message in `parseTransfer()`. Maybe I misunderstood, but I think domain *does* belong in the Transfer message, but when parsing it for the Connector, there's no point in also parsing and sending the domain

Reply:
True, we don't need to parse it now, but we should make sure to already define the length of the domain param. Otherwise if in the future this changes, we'll have to change the parsing of the arguments after it and that would not be a backwards compatible change.